README.md

```diff
@@ -36,13 +36,49 @@ Please visit the main [DataSketches website](https://datasketches.apache.org) fo
 
 If you are interested in making contributions to this site please see our [Community](https://datasketches.apache.org/docs/Community/) page for how to contact us.
 
+---
+
+## Java Support
+
+Datasketches memory currently supports Java 8 up to and including Java 13.
+
+In order to use the library in Java 9 and above, you must provide the following runtime arguments to the JVM:
+
+```shell
+    --add-opens java.base/java.nio=org.apache.datasketches.memory \
+    --add-opens java.base/jdk.internal.misc=org.apache.datasketches.memory \
+    --add-opens java.base/jdk.internal.ref=org.apache.datasketches.memory
+```
+
+For example, to run your local application with full compatibility for the Java module system, you might use the following command:
+```shell
+  $JAVA \
+    --module-path mods \
+    --add-opens java.base/java.nio=org.apache.datasketches.memory \
+    --add-opens java.base/jdk.internal.misc=org.apache.datasketches.memory \
+    --add-opens java.base/jdk.internal.ref=org.apache.datasketches.memory \
+    --module my.main.application.module
+```
+
+where `mods` is your module path and `my.main.application.module` is your own JPMS module:
+
+```java
+module datasketches.memory.multirelease.test {
+    requires org.apache.datasketches.memory;
+}
+```
+
+
 ---
 
 ## Build Instructions
 __NOTE:__ This component accesses resource files for testing. As a result, the directory elements of the full absolute path of the target installation directory must qualify as Java identifiers. In other words, the directory elements must not have any space characters (or non-Java identifier characters) in any of the path elements. This is required by the Oracle Java Specification in order to ensure location-independent access to resources: [See Oracle Location-Independent Access to Resources](https://docs.oracle.com/javase/8/docs/technotes/guides/lang/resources.html)
 
-### JDK8/Hotspot is required to compile
-This DataSketches component is pure Java and you must compile using JDK 8 with Hotspot.
+### JDK versions required to compile
+This DataSketches component is pure Java and requires the following JDKs to compile:
```

Comment:
I think you mean "one of the following".  (certainly not all of them simultaneously :) )

Reply:
I am not sure how to clarify this documentation - as the Maven toolchain build requires these three java versions to be present on the developer's machine in order to build the project successfully.  The end-user does not require all java versions to be present simultaneously.

Do you think that this needs further explanation in the README?